torch/_dynamo/bytecode_transformation.py

```diff
@@ -258,8 +332,25 @@ def maybe_pop_n(n):
     instructions[:] = output
     return added
 
+# from https://github.com/python/cpython/blob/v3.11.1/Include/internal/pycore_opcode.h#L41
+# TODO maybe use the actual object instead, interface from eval_frame.c?
```

Comment:
It's jut an int, but it would be nice to reuse, yea. I would give this its own PR for sure.